### PR TITLE
ibc: fix logging of timestamp timeouts

### PR DESCRIPTION
**Pull request by avahowell**

### CODE REVIEW

**Review by cratelyn — APPROVED** [review]

File: crates/core/component/ibc/src/component/packet.rs [hunk 1]
```diff
@@ -161,7 +161,7 @@ pub trait SendPacketRead: StateRead {
         if packet.timeout_timestamp <= chain_ts.nanoseconds() {
             anyhow::bail!(
                 "timeout timestamp {} is less than the latest timestamp on the counterparty {}",
-                packet.timeout_height,
+                packet.timeout_timestamp,
                 chain_ts,
```
Comment (zbuc) [inline crates/core/component/ibc/src/component/packet.rs:165]: can you change this to log `chain_ts.nanoseconds()` instead so the displayed types are the same?

**Review by zbuc — CHANGES_REQUESTED** [review]

**Comment by conorsch** [comment]
During interchaintest integration work, @zbuc noticed that the logging message was structured. Loosely related to #4638 & #4636.

**Review by zbuc — APPROVED** [review]

**Review by conorsch — APPROVED** [review]

**Comment by avahowell** [comment]
I just noticed that the timestamp used here is wrong, it should be using the consensus state timestamp. will fix

**Comment by conorsch** [comment]
Marking this as consensus-breaking after the fact, because the diff makes substantive changes to how timeouts are handled. Please remove the label if inappropriate.

**Pull request merged**